youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/PKYouboraAdsAdapter.java

```diff
@@ -297,12 +308,17 @@ private void addListeners() {
             sendReportEvent(event.eventType());
         });
 
+        messageBus.addListener(this, AdEvent.contentPauseRequested, event -> {
+            fireAdBreakStart();
+        });
+
         messageBus.addListener(this, AdEvent.contentResumeRequested, event -> {
             printEventName(event);
             if (isNullAdapter()) {
                 return;
             }
             fireStop();
+            fireAdBreakStop();
```

Comment:
Need to check the behaviour  when it is being fired

Reply:
If there is Preroll having 2 pod ads then `adbreakStart` and `adBreakStop` for each pod ad BUT if Midroll and Postroll has more than 1 pod ad then it does not matter. `adbreakStart` is being called when the first ad start and `adBreakStop` is being fired when all the ads are complete.